### DISCUSSION TITLE
Is it possible to trigger an Automatic Translation for multiple components?

### DISCUSSION THREAD

**Discussion by derkoe** [WeblateOrg/weblate#6259] [Q&A]
I can see the "Tools" > "Automatic Translation" menu when in a component and language but the button is not there when I am on the project level. Is it possible to trigger an Automatic Translation for multiple components in a project?

**Comment by luebbe** [comment]
You have to add the "Automatic translation" plugin to each component individually and configure it there.
Unfortunately there is no "Activate automatic translation for all components with the following identical settings" function in weblate.

**Reply by derkoe** [reply]
I was not talking about the extension.

Is talking about the function in the "Tools" menu
![image](https://user-images.githubusercontent.com/123199/124599380-e598ed80-de65-11eb-8904-7b3801d94893.png)

**Reply by luebbe** [reply]
I guess it's the same reason under the hood. The automatic translation extension can only be configured on component level, so it's not visible one level further up.

**Reply by nijel** [reply]
The biggest reason is that nobody implemented it there.

**Reply by rsr-cs** [reply]
We would also be interested in having this feature on the project and language level for multiple components, especially for large projects with many components. Do you have plans to implement it on this levels? Language level with higher priority.

**Reply by nijel** [reply]
@rsr-cs There is currently no plan for this, please discuss this with @orangesunny on customer care in case you are interested in funding this. Alternatively, you can use [automatic translation add-on](https://docs.weblate.org/en/latest/admin/addons.html#automatic-translation).
